src/redis-benchmark.c

```diff
@@ -1733,6 +1733,22 @@ int main(int argc, const char **argv) {
             free(cmd);
         }
 
+        if (test_is_selected("zadd")) {
+            char *score = "0";
+            if (config.randomkeys) score = "__rand_int__";
+            len = redisFormatCommand(&cmd,
+                "ZADD myzset:{tag} %s element:__rand_int__",score);
+            benchmark("ZADD",cmd,len);
+            free(cmd);
+        }
+
+        if (test_is_selected("zrem")) {
+            len = redisFormatCommand(&cmd,
```

Comment:
Should we proceed in the same manner as lrange* tests? Meaning that we always zadd before starting the zrem test to avoid zrem empty keys?
Lrange example:
https://github.com/redis/redis/blob/e5a50ed3c49c211b413d92b1f45f950c830c58c0/src/redis-benchmark.c#L1758

Reply:
Unsure - note the use of rand_int in member name so zrem is not assured to actually hit real members... unlike lrange.

Reply:
Yes, we should `zadd` members as much as possible before starting `zrem` test manually. To reduce the chance to delete inexistent members when do `zrem` test,  we should set `-n <requests>` is more much than`-r <keyspacelen>` when we execute `zadd` test, and set  `-n <requests>` is less than `-r <keyspacelen>` when do `zrem` test.  Just as @itamarhaber said, It is different from `lrange`, I also don't have a good idea to do `zrem` test automatically in code. 

The reason why I want to do zrem test is to know the performance  of deleting nodes of `skiplist`. But actually `zrem` means searching an element from dict if `zrem` inexistent members.

Maybe I needn't add `zrem` test just like there is no `hdel` test :(